tests/test_bucketlist.py

```diff
@@ -0,0 +1,182 @@
+from flask import json
+from base_class import TestBase
+from datetime import datetime
```

Comment:
Consider adding TESTS for cases where a user wants to create a duplicate bucketlist.